flexmeasures/ui/views/sensors.py

```diff
@@ -69,5 +69,7 @@ def get(self, id: int):
         return render_flexmeasures_template(
             "views/sensors.html",
             sensor_id=id,
+            event_starts_after=session.get("event_starts_after"),
```

Comment:
These settings should be used throughout the UI (for the same user), should they not? Then we should also set them in `render_flexmeasures_template`, like we use(d) to for the old `start_time` and `end_time` settings.

I can make that change.

Reply:
Correct. Thanks.